client/my-sites/checkout/composite-checkout/hooks/use-create-payment-complete-callback.tsx

```diff
@@ -191,6 +198,9 @@ export default function useCreatePaymentCompleteCallback( {
 				}
 			}
 
+			if ( isInEditor && isFocusedLaunch && ! hasEcommercePlan( responseCart ) ) {
+				return;
+			}
```

Comment:
Focused Launch is showing a thank you / success dialog directly in editor, after payment is complete. See https://github.com/Automattic/wp-calypso/pull/47808#issuecomment-755196691

Reply:
Sounds good. Could you add a comment to this line of code to mention that, just so in the future we'll know why this is here?